database/entity/0046-messages_tables/Message.ts

```diff
@@ -0,0 +1,29 @@
+import { BaseEntity, Column, Entity, JoinColumn, ManyToOne, PrimaryGeneratedColumn } from 'typeorm'
+import { Contribution } from '../Contribution'
+import { User } from '../User'
+
+@Entity('messages', { engine: 'InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci' })
+export class Message extends BaseEntity {
+  @PrimaryGeneratedColumn('increment', { unsigned: true })
+  id: number
+
+  @Column({ name: 'contribution_id', unsigned: true, nullable: false })
+  contributionId: number
+
+  @ManyToOne(() => Contribution, (contribution) => contribution.messages)
+  @JoinColumn({ name: 'contribution_id' })
+  contribution: Contribution
+
+  @Column({ name: 'user_id', unsigned: true, nullable: false })
+  userId: number
+
+  @ManyToOne(() => User, (user) => user.messages)
+  @JoinColumn({ name: 'user_id' })
+  user: User
```

Comment:
This may be a nice to have. What is the use case? Does it make sense to have all messages written by a user out of the context of the contribution?